doc/proposal/reflection/ProductTypes.md

```diff
@@ -89,28 +89,27 @@ This seems much simpler.
 
-To provide *extended tuple-like* access for all the types covered by [P0144R1] which support getting the size and the n<sup>th</sup> element, we need to define some kind of predefined operators `pt_size(T)`/`pt_get<N>(pt)` that could use the new *product-type-like* customization points.
+### Parameter packs
 
-#### Parameter packs
-
-We shouldn't forget parameter packs, that could be seen as being similar to product types. Parameter packs already have the `sizeof...(T)` operator. Some (see e.g. [P0311R0] and references therein) are proposing to have a way to explicitly access the n<sup>th</sup> element of a pack (a variety of possible syntaxes have been suggested). The authors believe that the same operators should apply to parameter packs and product types.
+We shouldn't forget parameter packs, which could be seen as being similar to product types. Parameter packs already have the `sizeof...(T)` operator. Some (see e.g. [P0311R0] and references therein) are proposing to have a way to explicitly access the n<sup>th</sup> element of a pack (a variety of possible syntaxes have been suggested). The authors believe that the same operators should apply to parameter packs and product types.
 
 # Proposal
 
-This paper proposes to define a new *product type* access, to cover the previous *extended tuple-like* access, on which [P0144R1] would be based. The user shall be able to customize his own types to see them as *product types* ([P0144R1] case 2).
+This paper proposes to define a new *product type* access, to cover the previous *extended tuple-like* access, on which [P0144R2] would be based. The user shall be able to customize his own types to see them as *product types* ([P0144R2] case 2).
 
-The *product type* access is based on two operators: one `pt_size(T) to get the size and the other `pt_get<N>(pt) to get the `N`<sup>th</sup> element of a product type instance `pt` of type `T`. The definition of these operators is based on the wording of structured binding [P0217R1].
+The *product type* access is based on two operators: one `pt_size(T)` to get the size and the other `pt_get<N>(pt)` to get the `N`<sup>th</sup> element of a product type instance `pt` of type `T`. The definition of these operators is based on the wording of structured binding [P0217R1].
 
 The user should of course, be able to customize his *product* types, as she already is able to do it for *tuple-like* types, but now it should define the member operations `product_type_size()` and `product_type_get<N>()`.
 
 The name of the operators `pt_size` and `pt_get` are of course subject to bike-shedding.
 This paper proposes the following names for the operators and the customization operations:
 
 * `pt_size(PT)` = `product_type_size(PT)`
-* `pt_get<N>(pt)` = `product_type_get(N,pt)`
+* `pt_get<N>(pt)` = `product_type_get<N>(pt)`
 
```

Comment:
I changed these to match the above (see end of first paragraph in above diff context); maybe that was an error on my part? Also because some might be confused how the return type can be variant based on `N`. If those changes are erroneous, please feel free to undo them.